pkg/models/trb_request.go

```diff
@@ -29,11 +29,40 @@ const (
 	TRBTFormalReview TRBRequestType = "FORMAL_REVIEW"
 )
 
-// TRBRequestStatus represents the types of TRBRequestStatus types.
+// TRBRequestStatus represents the types of TRBRequestStatus types
 type TRBRequestStatus string
 
 // These are the options for TRBRequestStatus
 const (
 	TRBSOpen   TRBRequestStatus = "OPEN"
 	TRBSClosed TRBRequestStatus = "CLOSED"
 )
+
+// TRBFeedbackStatus represents the types of TRBFeedbackStatus types
+type TRBFeedbackStatus string
```

Comment:
I don't see that this type is used anywhere; `TRBTaskStatus` is used by the GQL schema for feedback and consult. Is that intentional?

Reply:
Just removed this and pushed